p2p/src/authenticated/discovery/actors/peer/actor.rs

```diff
@@ -18,12 +23,13 @@ use rand::{CryptoRng, Rng};
 use std::{collections::HashMap, sync::Arc, time::Duration};
 use tracing::{debug, info};
 
-pub struct Actor<E: Spawner + Clock + ReasonablyRealtime + Metrics, C: PublicKey> {
+pub struct Actor<E: Spawner + Clock + ReasonablyRealtime + Metrics, C: PublicKey + Clone> {
```

Comment:
IIRF `PublicKey` is already `Clone`
(applies to line 46 as well)

Reply:
This was from a previous revision 🤦